### PR TITLE
Feat usn rebased

### PR DESCRIPTION
**Pull request by esaminu**
Adopted and rebased branch of #2568 

Also:
* Enabled flag on `testnet` for preview purposes
* Fix bugs from https://github.com/near/near-wallet/pull/2568#pullrequestreview-944202624

### CODE REVIEW

**Review by Patrick1904 — COMMENTED** [review]
The UI/UX is quite rough but I understand that we're going to ship a follow-up PR with UI improvements. Some obvious and minor fixes that I noted were:

- ‘Continue’ button enabled despite no NEAR or USN available to swap (results in failed TX)
- Ledger ‘processing’ modal keeps showing after swapping
- Since there’s no confirmation step before swapping (as FT flow), I think the button should say ‘Confirm’ vs ‘Continue’
- Confirmation screen copy says ‘You’ve swapped X NEAR To X USN’. Let’s align with FT flow: ‘You swapped X NEAR to X USN’
- Confirmation screen button says ‘Back to Swap’ - let’s say ‘Continue’ (align with FT flow) that goes to the dashboard (vs to swap page)
- Confirmation screen button says ‘To the main screen’ - let’s say ‘Swap More’

I think it's at least worth fixing the ledger issue before merging the PR

**Comment by andy-haynes** [comment]
I made some quality of life style/formatting changes after reviewing it last night, I think they'll help with long-term maintenance. We can roll them back if they break anything but I didn't notice any new problems when testing. Other than that the PR looks pretty good, the only thing I noticed was the leading `0` that persists even after entering a swap value, i.e. entering a `1` gets rendered in the input as `01`.

Once Patrick's feedback is addressed we should get this into testnet 🎉

File: packages/frontend/src/components/Routing.js [hunk 1]
```diff
@@ -199,7 +200,6 @@ class Routing extends Component {
         } = this.props;
 
         fetchTokenFiatValues();
-        this.startPollingTokenFiatValue();
```
Comment (andy-haynes) [inline packages/frontend/src/components/Routing.js:202]: Are we no longer polling the `/fiat` endpoint? Should this be feature flagged?

Comment (andy-haynes) [reply packages/frontend/src/components/Routing.js]: After talking with @Patrick1904 and @MaximusHaximus we've decided to revert this change and keep fiat polling in `Routing`

**Review by Patrick1904 — APPROVED** [review]
LGTM (for testnet)

**Pull request merged**